app/models/expanded_taxonomy.rb

```diff
@@ -0,0 +1,47 @@
+class ExpandedTaxonomy
```

Comment:
What do you think of creating a `Taxon` object with the response from the Publishing API in here and therefore replace calls like `taxon['content_id']` with `taxon.content_id`?

Something like this:

```
raw_root_content_item = Services.publishing_api.get_content(@content_id)
root_content_item = Taxon.new(raw_root_content_item)
```

I think it would improve readability in some places, especially in `app/views/taxonomies/show.html.erb` where we have quite a few value calls to a hash.